src/app/components/figure/figure-license-editor.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { TextField, IconButton } from '@material-ui/core';
+import { withStyles, WithStyles } from '@material-ui/core/styles';
+import { Node as ProsemirrorNode } from 'prosemirror-model';
+import DeleteIcon from '@material-ui/icons/Delete';
+
+import { FIGURE_LICENSE_CC0, FIGURE_LICENSE_SELECT_OPTIONS, FigureLicense } from 'app/models/figure-license';
+import { RichTextEditor } from 'app/components/rich-text-editor';
+import { NodeEditor } from 'app/components/node-editor/node-editor';
+import { useFigureLicenseEditorStyles } from 'app/components/figure/styles';
+import { Select } from 'app/components/select';
+import { NodeSelection } from 'prosemirror-state';
+
+interface FigureLicenseEditorProps extends WithStyles<typeof useFigureLicenseEditorStyles> {
+  node: ProsemirrorNode;
+  offset: number;
+  index: number;
+  classes: Record<string, string>;
+}
+
+class FigureContentEditorComponent extends NodeEditor<FigureLicenseEditorProps> {
+  render() {
+    const license: FigureLicense = this.props.node.attrs.licenseInfo as FigureLicense;
+    const classes = this.props.classes;
+    return (
+      <div>
+        <div className={classes.fieldsRow}>
+          <Select
+            className={classes.smallField}
+            name="licenseType"
+            placeholder="Please select"
+            canUnselect={false}
+            blankValue={undefined}
+            label="Licence type"
+            options={FIGURE_LICENSE_SELECT_OPTIONS}
+            value={license.licenseType}
+            onChange={this.handleFormChange}
+          />
+          {license.licenseType !== FIGURE_LICENSE_CC0 ? (
```

Comment:
I believe this can be simplified using an && without needing to perform a ternary statement - not sure if it would play nicely with TypeScript or not